scormxblock/scormxblock.py

```diff
@@ -178,6 +180,14 @@ def studio_submit(self, request, suffix=""):
             charset="utf8",
         )
 
+    def update_subdir_permissions(self):
+        # All sub-directories of the scorm package must have executable permissions for owner.
```

Comment:
@tehreem-sadat can we add a docstring to this method instead of inline comment and in that doctring we need to explain what this method does and add some background why we need this method.

Reply:
done.